service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/placement/AbstractPlacementControllerTest.java

```diff
@@ -5,23 +5,44 @@
 import uk.gov.hmcts.reform.fpl.model.PlacementConfidentialDocument;
 import uk.gov.hmcts.reform.fpl.model.PlacementSupportingDocument;
 import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.RespondentSolicitor;
 import uk.gov.hmcts.reform.fpl.model.common.Element;
 
 import static uk.gov.hmcts.reform.fpl.model.PlacementConfidentialDocument.Type.ANNEX_B;
 import static uk.gov.hmcts.reform.fpl.model.PlacementConfidentialDocument.Type.GUARDIANS_REPORT;
 import static uk.gov.hmcts.reform.fpl.model.PlacementSupportingDocument.Type.BIRTH_ADOPTION_CERTIFICATE;
 import static uk.gov.hmcts.reform.fpl.model.PlacementSupportingDocument.Type.STATEMENT_OF_FACTS;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testAddress;
 import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testChild;
 import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testDocumentReference;
-import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testRespondent;
 
 class AbstractPlacementControllerTest extends AbstractCallbackTest {
 
     final Element<Child> child1 = testChild("Alex", "Brown");
     final Element<Child> child2 = testChild("George", "White");
 
-    final Element<Respondent> respondent1 = testRespondent("Emma", "Green", "mother");
-    final Element<Respondent> respondent2 = testRespondent("Adam", "Green", "father");
+    final Element<Respondent> mother =
+        element(Respondent.builder()
+            .party(RespondentParty.builder()
+                .firstName("Emma")
+                .lastName("Green")
+                .relationshipToChild("mother")
```

Comment:
What made you have the relationshipToChild as a string rather than an enum?

Reply:
i did not change it, it is existing model

Reply:
it was from the very begging free text in UI